group.yml

```diff
@@ -23,6 +23,18 @@ branch: rhaos-{MAJOR}.{MINOR}-rhel-7
 # When the buildroot is bumped for a new version of go, the RPM should land in this repository
 # and the image will need to be rebuilt.
 repos:
+  rhel-7-golang118-rpms:
+    conf:
+      extra_options:
+        module_hotfixes: 1
```

Comment:
this would just be confusing, there not being any modules in rhel7 :stuck_out_tongue: 
```suggestion
```